### PR TITLE
Release notes and highlights for 1.5.0

### PR DESCRIPTION
**Pull request by thbkrkr**
Release notes and highlights for the upcoming minor release.

### CODE REVIEW

File: docs/release-notes/highlights-1.5.0.asciidoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+New and notable changes in version 1.5.0 of {n}. See <<release-notes-1.5.0>> for the full list of changes.
+
+[float]
+[id="{p}-150-autoscaling-support"]
+==== Support for Autoscaling
+
+ECK 1.5.0 introduces experimental support for link:https://www.elastic.co/guide/en/elasticsearch/reference/7.11/xpack-autoscaling.html[Autoscaling].
+
+In this initial release, Autoscaling monitors the storage utilization of your Elasticsearch data nodes and the available memory capacity for your machine learning jobs. As your data grows, whether you’re expanding to new use cases or simply storing data for longer, autoscaling automatically adjusts resource capacity to ensure you can store your data, and that your machine learning jobs can execute — so you don’t have to worry about whether your deployment can support your requirements. Future releases will include autoscaling based on additional metrics and stack components (such as Kibana).
```
Comment (alaudazzi) [inline docs/release-notes/highlights-1.5.0.asciidoc:16]:
```suggestion
In this initial release, autoscaling monitors the storage usage of your Elasticsearch data nodes and the available memory capacity for your machine learning jobs. As your data grows, whether you’re expanding to new use cases or simply storing data for longer, autoscaling automatically adjusts resource capacity to ensure you can store your data, and that your machine learning jobs can execute — so you don’t have to worry about whether your deployment can support your requirements. Future releases will include autoscaling based on additional metrics and stack components (such as Kibana).
```

File: docs/release-notes/highlights-1.5.0.asciidoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+In addition, you can now control what ECK should do with the PersistentVolumeClaims if you delete the Elasticsearch cluster altogether through the new `volumeClaimDeletePolicy` attribute.
+
+The possible values are `DeleteOnScaledownAndClusterDeletion` and `DeleteOnScaledownOnly`. By default `DeleteOnScaledownAndClusterDeletion` is in effect, which means that all PersistentVolumeClaims are deleted together with the Elasticsearch cluster. However, `DeleteOnScaledownOnly` keeps the PersistentVolumeClaims when deleting the Elasticsearch cluster. If you recreate a deleted cluster with the same name and node sets as before, the existing PersistentVolumeClaims will be adopted by the new cluster.
+
+[float]
+[id="{p}-150-enterprisesearch-resource-version-promotion"]
+==== EnterpriseSearch resource version promotion
+
+EnterpriseSearch resources have graduated to version `v1`. Old resources must be migrated to the new version to work with ECK 1.5.0. Resources created using an older version will continue to run but the operator will not manage them.
```
Comment (alaudazzi) [inline docs/release-notes/highlights-1.5.0.asciidoc:32]:
```suggestion
EnterpriseSearch resources have graduated to version `v1`. Old resources must be migrated to the new version to work with ECK 1.5.0. Resources created using an older version continue to run but the operator will not manage them.
```

**Review by alaudazzi — APPROVED** [review]
Minor edits, LGTM.

**Comment by thbkrkr** [comment]
I added a note for the end of OCP 3.11 support.

File: docs/release-notes/1.5.0.asciidoc [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+[float]
+=== Enhancements
+
+* Add support for VolumeClaimDeletePolicies for Elasticsearch clusters {pull}4050[#4050] (issue: {issue}2328[#2328])
+
+[[deprecation-1.5.0]]
+[float]
+=== Deprecation
+
+* OpenShift 3.11 support is deprecated and will no longer be supported with ECK 2.0. ECK 1.8 will be the last version to support OpenShift 3.11.
```
Comment (alaudazzi) [inline docs/release-notes/1.5.0.asciidoc:36]:
```suggestion
* OpenShift 3.11 support is deprecated in ECK 2.0. ECK 1.8 is the last version that supports OpenShift 3.11.
```

**Review by alaudazzi — APPROVED** [review]
Suggested minor edits. LGTM.

File: docs/release-notes/1.5.0.asciidoc (hunk 3 above)
Comment (pebrc) [reply docs/release-notes/1.5.0.asciidoc:36]: Should this whole section about the OpenShift 3.11 not be in the highlights instead?

Comment (thbkrkr) [reply docs/release-notes/1.5.0.asciidoc:36]: Yes, it's quite questinable and we thought with @shubhaat that this was not a highlight. No problem to change one's mind if there is a strong opinion.

**Pull request merged**